eng/sendtohelix.proj

```diff
@@ -82,7 +84,33 @@
 
   </Target>
 
-  <Target Name="BuildHelixWorkItems" DependsOnTargets="CompressRuntimeDirectory">
+  <Target Name="BuildHelixCommands">
+    <!-- We need to do this in a target in order to be able to transform package restore sources item into a string -->
+    <PropertyGroup Condition="'$(HelixCommand)' == '' AND '$(IsPackageTesting)' == 'true'">
+      <HelixPreCommands>set DOTNET_CLI_TELEMETRY_OPTOUT=1;set DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1;set DOTNET_MULTILEVEL_LOOKUP=0</HelixPreCommands>
+
+      <HelixCommand>%HELIX_CORRELATION_PAYLOAD%\tools\dotnet.exe msbuild %HELIX_CORRELATION_PAYLOAD%\test.msbuild</HelixCommand>
+      <HelixCommand>$(HelixCommand) /t:RestoreAndTest</HelixCommand>
+      <HelixCommand>$(HelixCommand) /p:PackageTestProjectsDir=%HELIX_WORKITEM_PAYLOAD%</HelixCommand>
+      <HelixCommand>$(HelixCommand) /p:RestorePackagesPath=%HELIX_WORKITEM_PAYLOAD%\packages</HelixCommand>
+
+      <!-- %3B is ; literal, we use it for the command line to avoid trying to be parsed -->
+      <HelixCommand>$(HelixCommand) /p:RestoreSources=@(PackageTestRestoreSources -> '%(Identity)', '%3B')</HelixCommand>
+    </PropertyGroup>
+
+
+    <PropertyGroup Condition="'$(HelixCommand)' == ''">
```

Comment:
NIT: it is probably clearer if the condition here is $(IsPackageTesting) != 'true' that way someone reading the targets file knows when would HelixCommand would be empty.

Reply:
This was to make the property overridable, I can add an AND clause to the condition including $(IsPackageTesting) as part of the condition.

Reply:
I see, nah I think it's fine as is.